src/state/create-store.js

```diff
@@ -20,8 +20,12 @@ import type { Store } from './store-types';
 // We are checking if window is available before using it.
 // This is needed for universal apps that render the component server side.
 // Details: https://github.com/zalmoxisus/redux-devtools-extension#12-advanced-store-setup
+const isDevelopment: boolean = process.env.NODE_ENV === 'development';
+
 const composeEnhancers =
-  typeof window !== 'undefined' && window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__
+  isDevelopment &&
```

Comment:
`process.env.NODE_ENV !== 'production'`

Sometimes there are other env strings used that are not development such as 'test'